language/sentiment/neg.txt

```diff
@@ -0,0 +1,4 @@
+What was Hollywood thinking with this movie! I hated,
```

Comment:
Can you rename the `sentiment` dir to `resources`, for consistency with the other samples